charactersheet/charactersheet/services/common/account/xmpp_connection_service.js

```diff
@@ -154,10 +154,12 @@ function _XMPPService(config) {
                 console.log('Connecting.');
             }
         } else if (status === Strophe.Status.AUTHFAIL) {
+            Notifications.xmpp.error.dispatch(status);
```

Comment:
There is one already. "You've joined <party>". I don't think adding one for successfully connecting to XMPP is necessary.